src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -290,19 +286,98 @@ await requestContext.SendResult(new ValidateWindowsAccountResult()
                 {
                     int ret = Marshal.GetLastWin32Error();
                     string errorMessage = new Win32Exception(Marshal.GetLastWin32Error()).Message;
-                    await requestContext.SendResult(new ValidateWindowsAccountResult()
-                    {
-                        Success = returnValue,
-                        ErrorMessage = errorMessage
-                    });
-                    return;
+                    await requestContext.SendError(errorMessage);
+                }
+                else
+                {
+                    await requestContext.SendResult(true);
                 }
+            } 
+            else 
+            {
+                await requestContext.SendResult(true);
+            }
+        }
+
+        internal async Task HandleValidateNetworkFileShareRequest(
+            ValidateNetworkFileShareRequestParams parameters,
+            RequestContext<bool> requestContext)
+        {
+            if (!ValidateWindowsDomainUsername(parameters.Username))
+            {
+                await requestContext.SendError("Invalid user name format. Example: Domain\\username");
+                return;
+            }
+
+            if (!ValidateUNCPath(parameters.Username))
```

Comment:
`parameter.Path`? 
Not sure if it would make more sense to update `Path` field in param def to something like `UNCPath`, just my 2 cents